src/SFA.DAS.EmployerIncentives/Application/Commands/RegisterEmploymentCheck/RegisterEmploymentCheckCommandHandler.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using SFA.DAS.EmployerIncentives.Configuration;
+using SFA.DAS.EmployerIncentives.InnerApi.Requests.EmploymentCheck;
+using SFA.DAS.SharedOuterApi.Extensions;
+using SFA.DAS.SharedOuterApi.Infrastructure;
+using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.EmployerIncentives.Application.Commands.RegisterEmploymentCheck
+{
+    public class RegisterEmploymentCheckCommandHandler : IRequestHandler<RegisterEmploymentCheckCommand, RegisterEmploymentCheckResponse>
+    {
+        private readonly IInternalApiClient<EmploymentCheckConfiguration> _client;
+
+        public RegisterEmploymentCheckCommandHandler(IInternalApiClient<EmploymentCheckConfiguration> client)
+        {
+            _client = client;
+        }
+
+        public async Task<RegisterEmploymentCheckResponse> Handle(RegisterEmploymentCheckCommand command, CancellationToken cancellationToken)
```

Comment:
I thought the established APIM pattern was to call a service from the command handler, rather than the API client directly?